src/__tests__/meta/cache-headers.js

```diff
@@ -0,0 +1,49 @@
+import http from 'http';
+import {startTestServer, stopTestServer} from '../helpers';
+
+const context = {};
+
+beforeAll(() => {
+  return startTestServer(context);
+});
+
+describe('cache headers', () => {
+
+  it('should return cache-control public on cacheable page', () => {
+    return new Promise((resolve) => {
+      const server = context.runningInstance;
+      http.get(`http://localhost:${server.info.port}${server.aka('start')}`, res => {
```

Comment:
Maybe you could add the ````http://localhost:${server.info.port}${server.aka('start')}```` to to the helpers module as a function so that you don't have to do the same over and over again? Something like ```resolve('start')```.